Remove product-specific sections and change organization (#28)
pyansys/pyansys_landing: README.rst

PyAnsys
=======
Welcome to the PyAnsys project.

The PyAnsys project is a collection of Python packages that enable the use of
Ansys products through Python.

This project originated as a single package, ``pyansys``, and has
expanded to several main packages:

- `PyAEDT <https://aedtdocs.pyansys.com/>`__ : Pythonic interface to AEDT (Ansys Electronic Desktop)
- `PyDPF-Core <https://dpfdocs.pyansys.com/>`__ : Pythonic interface to DPF (Data Processing Framework) for building more advanced and customized workflows

> - `PyDPF-Post <https://postdocs.pyansys.com/>`__ : Pythonic interface to DPF's postproccessing toolbox for maninuplating and transforming simulation data

Reviewer: ```suggestion
- `PyDPF-Post <https://postdocs.pyansys.com/>`__ : Pythonic interface to DPF's postprocessing toolbox for manipulating and transforming simulation data
```

- `PyMAPDL <https://mapdldocs.pyansys.com/>`__ : Pythonic interface to MAPDL.

> - `PyMAPDL Reader <https://readerdocs.pyansys.com/>`__: Pythnonic interface to read legacy MAPDL result files (MAPDL 14.5 and later)

Reviewer: ```suggestion
- `PyMAPDL Reader <https://readerdocs.pyansys.com/>`__: Pythonic interface to read legacy MAPDL result files (MAPDL 14.5 and later)
```

- `PyPIM <https://pypim.docs.pyansys.com/>`__: Pythonic interface to communicate with the PIM (Product Instance Management) API
- `Granta MI BoM Analytics <https://grantami.docs.pyansys.com/>`__: Pythonic interface to Granta MI BoM Analytics services
- `Shared Components <https://shared.docs.pyansys.com/>`_: Shared software components to enable package interoperability and minimize maintenance

Much effort is underway to continue expanding and developing packages in the
`PyAnsys GitHub <https://github.com/pyansys/>`__ account. On the ``Issues`` page

> for each package , you can post issues and request new features. You can also email

Reviewer: ```suggestion
for each package, you can post issues and request new features. You can also email
```

=======

> Welcome to the PyAnsys Project!

Author: ```suggestion
Welcome to the PyAnsys project.
```

> The PyAnsys project is a collection of Python packages to enable using

Author: ```suggestion
The PyAnsys project is a collection of Python packages to enable the use of
```

> The PyAnsys project is a collection of Python packages to enable the use of

Author: ```suggestion
The PyAnsys project is a collection of Python packages that enable the use of
```

Ansys products through Python.

> This project originated as a single package, ``pyansys``, and has been

Author: ```suggestion
This project originated as a single package, ``pyansys``, and has
```

expanded to several main packages:

- `PyAEDT <https://aedtdocs.pyansys.com/>`__ : Pythonic interface to AEDT (Ansys Electronic Desktop)
- `PyDPF-Core <https://dpfdocs.pyansys.com/>`__ : Pythonic interface to DPF (Data Processing Framework) for building more advanced and customized workflows
- `PyDPF-Post <https://postdocs.pyansys.com/>`__ : Pythonic interface to DPF's postproccessing toolbox for maninuplating and transforming simulation data
- `PyMAPDL <https://mapdldocs.pyansys.com/>`__ : Pythonic interface to MAPDL.
- `PyMAPDL Reader <https://readerdocs.pyansys.com/>`__: Pythnonic interface to read legacy MAPDL result files (MAPDL 14.5 and later)
- `PyPIM <https://pypim.docs.pyansys.com/>`__: Pythonic interface to communicate with the PIM (Product Instance Management) API
- `Granta MI BoM Analytics <https://grantami.docs.pyansys.com/>`__: Pythonic interface to Granta MI BoM Analytics services

> - `Shared Components <https://shared.docs.pyansys.com/>`_: Shared software components to enable package interoperability and minimize maintenance.

Author: ```suggestion
- `Shared Components <https://shared.docs.pyansys.com/>`_: Shared software components to enable package interoperability and minimize maintenance
```
